Class 2 Exercises and Homework.js

```diff
@@ -58,5 +93,23 @@ const months = [
   'December'
 ];
 
+// Get year for log message
+dueDateYear = dueDateObj.getFullYear();
 
 // 10. log this value using console.log
+// Print due date log message for every month in the months array
+console.log('\nDue dates for all months of the year are: ');
+let monthIndex = 0 // month index
+
+for (let month of months) {
```

Comment:
Cool that you can do this loop, but we only need the previously-found due date as an HTML tag.